.github/workflows/security-scan.yml

```
@@ -0,0 +1,390 @@
+            backend/bandit-report.json
+            backend/semgrep-report.json
+            frontend-nextjs/npm-audit-report.json
+            frontend-nextjs/eslint-security-report.json
+            e2e/npm-audit-report.json
+          retention-days: 30
+
+  # Container image scanning
+  container-scan:
+    name: Container Security Scan
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)